.github/workflows/gen-encodings-deploy.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     steps:
       - name: Clone stampy-ui repository code
         uses: actions/checkout@v3
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
```

Comment:
aah, it recycled the credentials from this step 🤦 thanks for having the patience to investigate 🙇